lib/index.js

```diff
@@ -103,10 +103,14 @@ util.inherits(Encoder, Transform)
 Encoder.prototype._transform = function(message, enc, cont) {
   var length = new Buffer(this.opts.lengthSize)
   this.setLength(length, message.length)
-  this.push(Buffer.concat([length, message]))
+  this._pushFrameData([length, message])
   cont()
 }
 
+Encoder.prototype._pushFrameData = function(bufs) {
```

Comment:
Did you had any specific reason in mind for using `function(bufs)`  instead of e.g. `function(length, chunk)`?